.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,7 +1,7 @@
 ### All Submissions:
 
-* [ ] Have you followed the [WooCommerce Contributing guideline](https://github.com/woocommerce/woocommerce/blob/master/.github/CONTRIBUTING.md)?
-* [ ] Does your code follow the [WordPress' coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/)?
+* [ ] Have you followed the [WooCommerce Contributing guideline](https://github.com/ClassicPress-research/classic-commerce/blob/master/.github/CONTRIBUTING.md)?
```

Comment:
"WooCommerce Contributing guideline" should be "ClassicCommerce contributing guidelines".

Reply:
Thanks